apps/todoapp/src/app/todos/components/user-info/user-info.component.scss

```diff
@@ -1,8 +1,40 @@
 .user {
-  display: flex;
-  align-items: center;
+  display: table-row;
+}
+
+.user * {
+  display: table-cell;
+  padding: 1px 0;
 }
 
 .user p {
-  margin-right: 8px;
+  padding-left: 10px;
+}
+
+.user img {
+  padding-right: 10px;
+}
+
+.user a {
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 4px 5px,
+    rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px,
+    rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+}
+
+.user a:hover {
+  box-shadow: white 0px 4px 5px inset, white 0px -3px 3px inset;
+}
+
+.clickableUser {
+  text-align: center;
+}
+
+.clickableUser * {
+  display: inline-block;
```

Comment:
why not using flex?

Reply:
what i was trying to achieve was not working with flex

Reply:
flex is much more robust for things like this, so I would set display: flex in .clickableUser and u can remove .clickableUser *